evap/staff/views.py

```diff
@@ -87,9 +87,61 @@ def __init__(self):
             self.first_start = datetime(9999, 1, 1)
             self.last_end = date(2000, 1, 1)
 
+    # course counts
```

Comment:
remove please :D

Edit: I like the rest, so I'll approve when this is changed :)